about.html

```diff
@@ -0,0 +1,102 @@
+          <img src="./images/expobavio-logo.png" alt="Expobavio Logo">
+        </a>
+        <ul class="navbar-nav mb-2 mb-lg-0">
+          <li class="nav-item px-3">
+            <a class="nav-link" href="./about.html">About</a>
+          </li>
+          <li class="nav-item px-3">
+            <a class="nav-link" href="./index.html#program">Program</a>
+          </li>
+          <li class="nav-item px-3">
+            <a class="nav-link" href="./index.html#speakers">Speakers</a>
+          </li>
+          <li class="nav-item px-3">
+            <a class="nav-link" href="./index.html#sponsors">Sponsors</a>
+          </li>
+          <li class="nav-item px-3">
+            <a class="nav-link" href="#">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <main>
+    <section id="headline" class="container d-flex flex-column justify-content-center align-items-center">
+      <div class="col-md-9 d-flex flex-column align-items-center h-100">
+        <h1 class="text-center">EXPOBAVIO<br>2021</h1>
+        <hr class="mt-3 mb-5">
+        <div id="about-desc" class="main-description text-center px-5 py-3">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 
+          incididunt ut labore et dolore magna aliqua.<br>Ut enim ad minim veniam, quis nostrud 
+          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<br>Duis aute 
+          irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla 
+          pariatur.</p>
+          <p class="d-none d-lg-block">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 
+          incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud 
+          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+        </div>
+        <p id='contact-text' class="text-center">
+          Please contact us per Email for any further questions about Expobavio 2021!<br>
+          <a href="contact@expobavio.com.ar"><strong>contact@expobavio.com.ar</strong></a> >
+        </p>
+      </div>
+    </section>
+    <section id="past-expos" class="white-sec container">
+      <div class="d-flex flex-column align-items-center">
+        <img src="./images/expobavio-logo.png" alt="Expo Logo" id="big-logo" class="pt-5 pb-2">
+        <h2>See the past Expobavio annual exhibitions</h2>
+        <hr class="my-2">
+        <p>Take a look at the last two expos which also took place in our Agricultural High School</p>
+        <div class="row mt-4 mb-5">
+          <div class="past-item col-md-5 m-2">
+            <div id="expo18" class="inner-past-item d-flex justify-content-center align-items-center">
+              <h3>2018</h3>
+            </div>
+          </div>
+          <div class="past-item col-md-5 m-2">
+            <div id="expo19" class="inner-past-item d-flex justify-content-center align-items-center">
```

Comment:
- [ ] Images in this section should be aligned to the center in mobile resolution. They aligned to the left and kind of looks out of the document flow 
![Screenshot from 2021-09-09 22-07-23](https://user-images.githubusercontent.com/1642014/132749596-72bfb8f0-e31b-4b80-9fd5-2615f484485b.png)